### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by quasilyte**
Commits:
```
linter: simplify parsePHPDocVar method
linttest: remove BadModifiers test
phpgrep: fix input param accidental shadowing
meta: remove commented-out debug print stmt
all: fix style issues found by linters
```

### CODE REVIEW

File: src/linter/root.go [hunk 1]
```diff
@@ -627,11 +627,7 @@ func (d *RootWalker) enterPropertyList(pl *stmt.PropertyList) bool {
 
 		nm := p.Variable.Name
 
-		typ, errText := d.parsePHPDocVar(p.PhpDocComment)
-		if errText != "" {
-			d.Report(p.Variable, LevelWarning, "phpdocLint", errText)
-		}
-
+		typ := d.parsePHPDocVar(p.PhpDocComment)
```
Comment (YuriyNasretdinov) [inline src/linter/root.go]: Why remove this check?

File: src/linttest/oop_test.go [hunk 2]
```diff
@@ -104,23 +104,6 @@ $_ = WithProps::$int;
...
 }
 
-func TestBadModifiers(t *testing.T) {
-	t.Skip("Should be handled by other check, like keywordCase from #138")
-
-	test := linttest.NewSuite(t)
-	test.AddFile(`<?php
-class Foo {
-  /***/
-  PUBLIC Static function f() {}
```
Comment (YuriyNasretdinov) [inline src/linttest/oop_test.go]: Why remove this test?

Comment (quasilyte) [reply src/linttest/oop_test.go]: It's was always skipped and issues are covered in keywordCase test.

File: src/linter/root.go (hunk 1 above)
Comment (quasilyte) [reply src/linter/root.go]: It wasn't checking anything, parsePHPDocVar always returns an empty string as an error text.

**Pull request merged**